examples/amd-azure/requirements.txt

```diff
@@ -0,0 +1 @@
+py-libnuma>=1.2
```

Comment:
Could we also add `deepsparse` here so that users won't have to first `pip install deepsparse` manually and then also install using `requirements.txt`;

if `deepsparse` is added, just installing from `requirements.txt` would be enough